<launch>

   <!-- Launch the sample image -->
   <node pkg="workshop" type="image_publisher.py" name="image_publisher_node" output="screen" />

   <!-- Launch crosswalk detector -->
   <node pkg="workshop" type="barrel_detector_vision.py" name="barrel_detector_node" output="screen" />

   <!-- Load dynamic reconfigure -->
   <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_console" args="-t"/>

   <!-- Dynamic Reconfigure -->
   <node name="dynamic_reconfigure_barrel" pkg="dynamic_reconfigure" type="dynparam" args="load /barrel_detector_node $(find workshop)/params/barrel_detector_vision.yaml"/>

</launch>
